code/pizzabot.js

```diff
@@ -1,7 +1,93 @@
+// Variables
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
 
+// Validation
+const pizzaList = [
+  "Vegetarian Pizza",
+  "Hawaiian Pizza",
+  "Pepperoni Pizza",
+  "Vegetarian",
+  "Hawaiian",
+  "Pepperoni",
+  "vegetarian pizza",
+  "hawaiian pizza",
+  "pepperoni pizza",
+  "vegetarian",
+  "hawaiian",
+  "pepperoni", ]
+
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+
+const validateOrderName = (orderName) => {
+  if (pizzaList.includes(orderName)) {
+    return true;  
+  } else {
+    return false; 
+  }
+};
+
+
+const calculateTotalCost = (orderQuantity, pizzaPrice) => {
+  const totalCost = (orderQuantity * pizzaPrice);
+  return totalCost;
+};
+
+const calculateCookingTime = (orderQuantity) => {
+
+  if (orderQuantity >= 1 && orderQuantity <= 2 ) {
+    return "10 min";
+
+  } else if (orderQuantity >= 3 && orderQuantity <= 5) {
+    return "15 min";
+
+  } else { 
+    return "20 min";
+  }
+
+};
+
+// startOrder is called when the user clicks the "Start order" button. 
+const startOrder = () => {
+  const botMessage = `Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}`
+  alert(botMessage);
+
+  askForPizza();
+}
```

Comment:
I really like that `startOrder` and `askForPizza` are two different functions here.  Great thinking.  Whatever line of thought led to this, pay attention to it in the future 👍